frontend/src/libs/components/link/link.tsx

```diff
@@ -32,7 +34,7 @@ const Link: React.FC<Properties> = ({
   );
 
   return (
-    <NavLink to={to} className={handleGetClassesByLinkState}>
+    <NavLink to={to} className={handleGetClassesByLinkState} onClick={onClick}>
```

Comment:
I’m not sure that we should add `onClick` to link, maybe it will be better to use a `Button` in the place where you need that?